common/views/components/SearchFiltersMobile/SearchFiltersMobile.tsx

```diff
@@ -307,9 +308,13 @@ const SearchFiltersMobile: FunctionComponent<SearchFiltersSharedProps> = ({
           focusable.setAttribute('tabIndex', '-1')
         );
 
-      openFiltersButtonRef &&
-        openFiltersButtonRef.current &&
-        openFiltersButtonRef.current.focus();
+      if (!firstRender.current) {
+        openFiltersButtonRef &&
+          openFiltersButtonRef.current &&
+          openFiltersButtonRef.current.focus();
+
+        firstRender.current = false;
+      }
```

Comment:
Is [this](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/hooks/useSkipInitialEffect.ts) the same thing?